.github/CONTRIBUTING.md

```diff
@@ -53,16 +42,8 @@ not already a CLA on file for your GitHub account. Once we have received the sig
 
 You can review details of the agreement at https://cla.opensource.microsoft.com.
 
-## Housekeeping
-
-Your pull request should: 
-
-* Include a description of what your change intends to do
-* Be a child commit of a reasonably recent commit in the **main** branch 
-    * Requests need not be a single commit, but should be a linear sequence of commits (i.e. no merge commits in your PR)
-* Have clear commit messages 
-    * e.g. "Refactor feature", "Fix issue", "Add tests for issue"
-
-## Running the Tests
+### Pull Requests
+For pull requests, please use the [pull request template](https://github.com/microsoft/AdaptiveCards/blob/main/.github/pull_request_template.md) we provide. Do test your fixes manually (PR support/Buddy builds at this point are not supported but we are working on enabling it) before putting up your change for review. All unit tests should be passing for a PR to be committed into main. Someone from the AdaptiveCards v-team will also have to review your change before the change is allowed to be merged in. They may ask questions for more information or ask you to change things. Be sure to respond to their comments and push additional changes to the branch if they ask you to modify things before they sign off.
 
-*Coming soon*
+#### Stale Pull Request Policy
+We mark pull requests as stale after 5 days of inactivity on it and call your attention to it via our fabricbot, and assign the pull request to be reviewed by a member of the Adaptive Cards team formally and should be able to help shepherd the change in. Our team's pull request SLA typically is ~2 business days but please do call us out on this if you see us slipping (we still are a very small team). The bot will automtically close the pull request and notify you after 30 days of continuous inactivity on it. Follow up's would likely be a case-by-case basis depending on why the PR aged out to begin with.
```

Comment:
the bot appears as `msftbot`, so we should probably refer to it as such externally :)